dxr/static_unhashed/css/dxr.css

```diff
@@ -190,13 +188,19 @@ code a {
     background-color: transparent;
 }
 
-/* results */
+/* promoted stuff, on the top */
+.promoted-container {
+    padding: 0.05em 1.0em 0;
```

Comment:
Why don't you see if you can bring it back and get it to work with the margin. The 2 sets of results blend together right now—but not quite enough to be consistent, with their differing indents for the icons—so it seems accidental.